### ISSUE TITLE
Use TensorFlow input ops to feed data into the network

### ISSUE THREAD

**Issue by reuben** [mozilla/DeepSpeech#109]

**Comment by kdavis-mozilla** [comment]
Just ran a test on the [not_working](https://github.com/mozilla/DeepSpeech/tree/not_working) branch and introduced placeholders there instead of the
```python
batch_x, batch_y, batch_seq_len = ted_lium.train.next_batch(batch_size)
```
calls. 

Doing so turned the [not_working](https://github.com/mozilla/DeepSpeech/tree/not_working) branch into a working branch. Hence, this issue will solve issue #62

**Comment by kdavis-mozilla** [comment]
Fixed with commit 3ad4f7c
